app/uk/gov/hmrc/teamsandrepositories/schedulers/GithubRatelimitMetricsScheduler.scala

```diff
@@ -64,15 +65,22 @@ class GithubRatelimitMetricsScheduler @Inject()(
       }.toMap
   }
 
-  val source: MetricSource =
+  private val source: MetricSource =
     new MetricSource {
       def metrics(implicit ec: ExecutionContext): Future[Map[String, Int]] =
         metricsDefinitions.toList.traverse { case (k, f) => f().map(i => (k, i)) }.map(_.toMap)
     }
 
-  val metricOrchestrator = new MetricOrchestrator(
+  private val lockService: LockService =
+    LockService(
+      lockRepository = mongoLockRepository,
+      lockId         = "metrix-lock",
```

Comment:
calling this `metrix` might be a little confusing. `github-rate-limit-metrics-lock` or something like that might be better